script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -100,7 +100,7 @@ Whoa, snagged a leftover curry bread!"
 It's no wonder that Senpai is surprised.
 I wonder how long he had been sneaking up on us, but he reached from behind the bench, proudly snatched a piece of curry bread, and ate it in one bite.
 <Page34>
-"I see, this one's good. It's really good. It tastes more like curry than anything else. It's too much of a treat for me when I just woke up on an empty stomach!
+"I see, this one's good. It's really good. It tastes more like curry than anything else. It's too much of a treat for me when I've just woken up on an empty stomach!
```

Comment:
Is "...It's too much of a treat for me when I've just woken up on an empty stomach!" accurate? Or does he mean to say something closer to:

"Though it's a bit much for my stomach when I've just woken up"?

Reply:
I think they mean basically the same thing. It's like when you describe something as "too rich", but literally:
> ご馳走すぎます
too much of a treat